docs/nodes/versions.js

```diff
@@ -0,0 +1,8 @@
+const versions = Object.freeze({
```

Comment:
I think this file is obsolete now.

Reply:
hm, I think it may be good to lock in golang here for now, wdyt?

Reply:
changed to `constants.js` for different versions of golang